src/Upgrader/Business/Collection/UpgraderCollection.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Business\Collection;
+
+use ArrayAccess;
+use ArrayIterator;
+use Countable;
+use IteratorAggregate;
+use Upgrader\Business\Exception\UpgraderException;
+
+abstract class UpgraderCollection implements Countable, IteratorAggregate, ArrayAccess
```

Comment:
Could you please clean up the class as we don't use most of the functionality.

Reply:
We need it for use collections inside for each construction 
discussed